backend/alembic/versions/2025_01_07_2208-98a16d1ef6be_adding_new_table_for_canhistory.py

```diff
@@ -27,7 +27,7 @@ def upgrade() -> None:
     sa.Column('ops_event_id', sa.Integer(), autoincrement=False, nullable=True),
     sa.Column('history_title', sa.String(), autoincrement=False, nullable=True),
     sa.Column('history_message', sa.Text(), autoincrement=False, nullable=True),
-    sa.Column('timestamp', sa.String(), autoincrement=False, nullable=True),
+    sa.Column('timestamp', sa.DateTime(), autoincrement=False, nullable=True),
```

Comment:
@rajohnson90 this doesn't actually make the change in alembic though. you'd need to have a new alembic version file that explicitly changes this column type from string/varchar to datetime if that's what's needed. I verified this in the azure environments that this is not getting changed to datetime.